sphinx/source/docs/dev_guide/documentation.rst

```diff
@@ -3,39 +3,104 @@
+point.
 
-Sphinx_ is used to generate HTML documentation. Due to the innate dependency
-of Bokeh on JavaScript, no other output formats are supported by the official
-build configuration. This section describes how to use Sphinx to build the
-Bokeh documentation from source.
+This section describes Bokeh's `style guidelines`_ for contributing to the
+documentation. This section also includes details about how to `build`_ and
+`edit`_ the documentation in your local development environment.
 
-Install Requirements
-~~~~~~~~~~~~~~~~~~~~
+.. _`style guidelines`:
 
-In order to build the docs from source, it is recommended that you first
-follow the instructions in :ref:`devguide_setup`.
+Documentation style guidelines
+------------------------------
 
-Some of the Bokeh examples in the documentation rely on sample data that is
-not included in the Bokeh GitHub repository or released packages, due to
-their size.
+Bokeh's documentation uses the `Google developer documentation style guide`_.
 
-Once Bokeh is installed, the sample data can be obtained by executing the
-following command at a Bash or Windows prompt:
+If your contribution includes substantial edits or additions, please
+familiarize yourself with Google's style guide. A simple way to get started
+is using Google's free `technical writing courses`_.
+
+Principles of good style
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Follow these basic guidelines in all your writing for Bokeh:
+
+.. vale off
+
+* Be aware of **sentence length**. Try to avoid sentences that require more
+  than two commas. Consider breaking up longer sentences. You could also use
+  bulleted or numbered lists instead.
+* Avoid jargon or uncommon words and phrases. Keep in mind that many users of
+  Bokeh don't use English as their primary language.
+* Use **active voice** instead of passive voice whenever possible.
+* Address the reader in the **second person ('you')**. Avoid using the first
+  person ('we') or the third person ('the user').
+* Use **American English** spelling and grammar (refer to `Merriam-Webster`_ for
+  consistent spelling)
+* Use **sentence case for headlines** (capitalize words like in a regular
+  sentence, but do not use any punctuation at the end).
+* Use **serial commas**, also known as Oxford Commas.
+* Write in a way that is inclusive_ and accessible_.
+* Activate the **spell checker** in your development environment.
+
+.. vale on
+
+Refer to the `Google developer documentation style guide`_ for more detailed
+information.
+
+Commonly used terms
+~~~~~~~~~~~~~~~~~~~
+
+These are some commonly used terms and phrases and their spellings used
+throughout the narrative documentation:
+
+.. csv-table::
+   :header: "Term", "Details"
+   :widths: 25, 75
+
+   "Bokeh, BokehJS", "Always capitalize Bokeh and BokehJS"
+   "JavaScript", "Capitalize both 'J' and 'S'"
+   "Jupyter notebook", "Capitalize Jupyter, but not notebook"
+   "Python", "Always capitalize Python (the language)"
+   "[TBD]", "[TBD]"
```

Comment:
I'd appreciate some more ideas for what else to include here. @bryevdv mentioned some link he was aware of regarding 'conda', for example.

Reply:
I don't have any other suggestions offhand, I think it's OK to start with a short list and expand it later as there are things we observe and decide to codify more carefully

Reply:
Absolutely! (Leaving this conversation open for now, in case anybody else has things to add)